ext/pdo_pgsql/tests/bug_last_insert_id.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+currval() vs lastval()
+PDO PgSQL Bug #1134 [BUG] New record, PostgreSQL and the Primary key https://github.com/phalcon/cphalcon/issues/1134
+Fix PHP original PDO_PGSQL::lastInsertId https://github.com/doctrine/dbal/pull/2329
+--SKIPIF--
+<?php
+if (!extension_loaded('pdo') || !extension_loaded('pdo_pgsql')) die('skip not loaded');
+require dirname(__FILE__) . '/config.inc';
+require dirname(__FILE__) . '/../../../ext/pdo/tests/pdo_test.inc';
+PDOTest::skip();
+?>
+--FILE--
+<?php
+
+require dirname(__FILE__) . '/../../../ext/pdo/tests/pdo_test.inc';
+
+$config['ENV']['PDOTEST_DSN'] = 'pgsql:host=localhost port=5432 dbname=test user=postgres password=postgres';
+$config['ENV']['PDOTEST_USER'] = 'postgres';
+$config['ENV']['PDOTEST_PASS'] = 'postgres';
+
+$db = PDOTest::factory();
```

Comment:
Please use $db = PDOTest::test_factory(dirname(__FILE__) . '/common.phpt'); so all the tests can be run on the same DB instance.

Thanks.